client/components/carousel.jsx

```diff
@@ -41,14 +43,18 @@ export default class Carousel extends React.Component {
       iterator === 0
         ? this.setState({ iterator: recipes.length - 1 })
         : this.setState({ iterator: iterator - 1 });
+    } else {
+      setLocalStorage('user-full-recipe', recipes[iterator]);
+      const updatedTitle = recipes[iterator].title.split(' ').join('+');
+      window.location.hash += `recipe?title=${updatedTitle}`;
```

Comment:
since we're not really using this title in the recipe page, it doesn't make much sense to put it in the hash. instead, it would be better to put the id in the hash and then **fetch** that specific recipe from the api, so that the user's back and forward buttons work.